packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -150,26 +192,26 @@ export interface ComboBoxProps
   /**
    * Message which is displayed if the value is invalid.
    */
-  invalidText?: ReactNodeLike;
+  invalidText?: ReactNode;
```

Comment:
Here we used to have `ReactNodeLike` but maybe we could change to `string`, since we are using only text.

Reply:
It's a node down in the proptypes, so I think this is fine 👍